src/ProjectManager.cs

```diff
@@ -157,20 +157,33 @@ public string GetPtusaProjectsPath()
                     StreamWriter sr = new StreamWriter(path, true);
                     sr.WriteLine("[path]\nfolder_path=");
                     sr.Close();
-                    sr.Flush();
+                    sr.Flush();         
                 }
                 PInvoke.IniFile iniFile = new PInvoke.IniFile(path);
 
                 // Считывание и возврат пути каталога проектов
-                string projectsFolder = iniFile.ReadString("path", "folder_path", "");
-                if (projectsFolder.Last() == '\\')
+                string projectsFolders =
+                    iniFile.ReadString("path", "folder_path", "");
+                string projectsFolder = "";
+
+                string[] projectsFoldersArray = projectsFolders.Split(';');
+                for (int i = 0; i < projectsFolder.Length; i++)
                 {
-                    return projectsFolder;
+                    if (projectsFoldersArray[i] != "")
+                    {
+                        projectsFolders = projectsFoldersArray[i];
```

Comment:
```suggestion
                        projectsFolder = projectsFoldersArray[i];
```

Допущена ошибка в логике. Мы работаем с ProjectFolder, но никак не с ProjectFolders т.к ProjectFolders содержит все наши пути в виде строки (который мы в дальнейшем преобразовали в массив projectsFoldersArray с помощью метода Split).